packages/react-components/src/components/utils/filtrer-tree/table-cell.tsx

```diff
@@ -12,9 +12,10 @@ export class TableCell extends React.Component<TableCellProps> {
     }
 
     render(): React.ReactNode {
+        const style = this.props.index === 0 ? { display: 'flex' } : undefined;
```

Comment:
Remove flex. Use instead: `const style = this.props.index === 0 ? { margin: '0 5px' } : undefined;`

Reply:
I didn't find that this margin was necessary.